src/components/pill-selector-dropdown/mapInputValuesToOptionsParser.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import uniqBy from 'lodash/uniqBy';
+import escapeRegExp from 'lodash/escapeRegExp';
+
+import parseCSV from '../../utils/parseCSV';
+
+import type { Option } from './flowTypes';
+
+// Custom input parser for the PillSelectorDropdownField that automatically maps
+// a list of comma separated values to their respective Option objects. The parser
+// will try to find exact but case-insenstive matches using the option's displayText
+// property. If a token does not match displayText, the parser will also try to match
+// the string representation of the value property. This last match is useful for country
+// codes and other similar use cases.
+function mapInputValuesToOptionsParser(
```

Comment:
probably remove the suffix Parser. mapInputValuesToOptions should suffice.

Reply:
Also, should we think making this the default? As in having this logic be called by parsePills() in PillSelectorDropdown?

Since it feels this may be useful for other ones too, like collab pill selector. I copy paste Foo, Bar, Bam@email.com and it should use those options from the collab drop down if available?

Reply:
"making this the default" or guarded by a prop

Reply:
Yes I think this is useful enough to be the default but I didn't want to change the current behavior. This parser only works in the context of `PillSelectorDropdownField` which I believe is not widely used yet though. Do you want me to add a prop that turns on this parser?

Reply:
Hmm, I would say just make it default for the Field for now. Can re-visit later.